frontend/src/index.css

```
@@ -1,10 +1,15 @@
+@import "tailwindcss";
+@import "tw-animate-css";
+@import "@fontsource-variable/inter";
+@import "shadcn/tailwind.css";
+
+@custom-variant dark (&:is(.dark *));
```

Stylelint (17.7.0)

[error] Unexpected unknown at-rule "@custom-variant" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)